### PR TITLE
Added OC config and removed options for ROS rectification / calibrati…

### PR DESCRIPTION
**Pull request by belt-sander**
This PR should address the following:

- I have removed the option to look up the calibration files as we are not going to be using the ROS / OpenCV rectification nodes
- I have defaulted all `calibrated` args to `False`
- I have added support for the OC stereo pair so that we can now collect data from these cameras using `roslaunch spinnaker_camera_driver kinetic_oc_stereo.launch` 
- I have renamed the stereo camera topics to be `upper` and `lower` as opposed to `left` and `right`

### CODE REVIEW

**Comment by ctkinetic** [comment]
Do we want to use upper and lower here or primary and secondary? see also [this slack discussion](https://kinetic-jn76634.slack.com/archives/C052KQT4KU2/p1683309502825429)

**Comment by belt-sander** [comment]
Added `primary` and `secondary` names

File: spinnaker_camera_driver/rviz/cam_test.rviz [hunk 1]
```diff
@@ -24,132 +24,41 @@ Panels:
...
-      Style: Points
-      Topic: /stereo/points2
-      Unreliable: false
-      Use Fixed Frame: true
-      Use rainbow: true
-      Value: false
     - Class: rviz/Image
       Enabled: true
-      Image Topic: /stereo/color/image_color
+      Image Topic: /stereo/upper/image_raw
```
Comment (ctkinetic) [inline spinnaker_camera_driver/rviz/cam_test.rviz:49]: Do you want to update the RViz file to use primary/secondary topics?

File: .idea/.gitignore
Comment (ctkinetic) [inline .idea/.gitignore:1]: I recommend removing `.idea` files and adding `idea` to `.gitignore`.  That might be a separate PR though

**Review by ctkinetic — COMMENTED** [review]
It is still not clear if the left or the right should be primary, so I am hesitant to approve.  Also, where do the `Image` topics get set?

File: spinnaker_camera_driver/rviz/cam_test.rviz (hunk 1 above)
Comment (belt-sander) [reply spinnaker_camera_driver/rviz/cam_test.rviz:49]: I deleted those topics from the `.rviz` config because it isn't needed here. The images from the primary / secondary topics are displayed in the `manager` rviz instance. 

I just view this `cam_test.rviz` as something that should be called for dumb tests

File: .idea/.gitignore
Comment (belt-sander) [reply .idea/.gitignore:1]: Copy. Good call out will add.

**Comment by belt-sander** [comment]
> It is still not clear if the left or the right should be primary, so I am hesitant to approve. Also, where do the `Image` topics get set?

The topic names get set as a combination of the namespace params. See a snip below from the `kinetic_oc_stereo.launch` file

```  <group ns="$(arg camera_name)">
      <group ns="secondary">
      <node pkg="nodelet" type="nodelet" name="camera_nodelet_manager" args="manager" cwd="node" output="screen"/>```

**Review by ctkinetic — APPROVED** [review]
LGTM
